_data/osdf_namespace_metadata/ospool-ap20.yaml

```diff
@@ -1,15 +1,15 @@
 description: |
-
-organization: # Name of the organization
-dataVisibility: # public, or private
-size: # In bytes so I can parse how I want
+  Data staging area for OSPool Access Point, AP20
+organization: University of Chicago # Name of the organization
+dataVisibility: private # public, or private
+size: 18000000000000 # In bytes so I can parse how I want
 bytesXferd: # In bytes so I can parse how I want
-url: # Organization URL
-fieldOfScience: # Top level Field of Science pulled from here https://nces.ed.gov/ipeds/cipcode/browse.aspx?y=55
+url: https://osg-htc.org/services/ospool/ # Organization URL
+fieldOfScience: 30) MULTI/INTERDISCIPLINARY STUDIES. # Top level Field of Science pulled from here https://nces.ed.gov/ipeds/cipcode/browse.aspx?y=55
```

Comment:
Elsewhere we've been doing only the human-readable string

Reply:
(without the period)

Reply:
Done, and migrated from all caps per BrianB's request